### ISSUE TITLE
helm upgrade fails dynatrace-operator-0.2.2

### ISSUE THREAD

**Issue by cvdsande** [Dynatrace/helm-charts#114]
Hi, I've been working on some automation that checks for new releases, and essentially runs "helm upgrade" if there's a new release.  I was excited to see a new release today so I could test the automation.

Unfortunately the upgrade fails.

Steps: (Installed version 0.2.1 using this identical command except for --version)
```
helm upgrade dynatrace-operator dynatrace/dynatrace-operator \
  --install \
  --create-namespace \
  -n dynatrace \
  -f values.yaml \
  --version 0.2.2 \
  --set networkZone=$CLUSTER \
  --set apiToken=$DT_API_TOKEN \
  --set paasToken=$DT_PAAS_TOKEN \
  --set-string 'classicFullStack.env[0].name=ONEAGENT_ENABLE_VOLUME_STORAGE,classicFullStack.env[0].value=true'
```

Error produced

Error: UPGRADE FAILED: rendered manifests contain a resource that already exists. Unable to continue with update: CustomResourceDefinition "dynakubes.dynatrace.com" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "dynatrace-operator"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "dynatrace"

values.yaml
```
apiUrl: "https://<redacted>.live.dynatrace.com/api"
createPodSecurityPolicies: false

classicFullStack:
  tolerations:
    - key: "app"
      operator: "Exists"
```

**Comment by cvdsande** [comment]
Just to make sure, I tested with with a command similar to what's recommended in the README.

```
 $ helm upgrade dynatrace-operator dynatrace/dynatrace-operator -n dynatrace --version 0.2.2 --reuse-values

Error: UPGRADE FAILED: rendered manifests contain a resource that already exists. Unable to continue with update: CustomResourceDefinition "dynakubes.dynatrace.com" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "dynatrace-operator"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "dynatrace"
```

**Comment by DTMad** [comment]
Hey @cvdsande,

thank you very much for bringing that to our attention!
For this release we had to move the CRDs into the template folder and therefore the upgrading does not work as it used to.

Helm does not update nor set the labels needed (see error message) nor the annotations to applications within the crds folder. Therefore I now created a patch file to patch the existing CRD so it's possible to upgrade it normally afterwards.

I updated the release and the corresponding update command:
https://github.com/Dynatrace/helm-charts/releases/tag/dynatrace-operator-v0.2.2

For the next releases your automation should now work ;)

Thanks,
Marco
